src/api/Item.js

```diff
@@ -333,7 +334,9 @@ class Item extends Base {
             this.successCallback = successCallback;
             this.errorCallback = errorCallback;
 
-            this.validateRequest(id, permissions);
+            // if we use the default access level, we don't need permission to set the access level
+            const skipCanSetShareAccess = access === undefined;
```

Comment:
what if access is null?

Reply:
if access is null, it is treated as a "remove shared link" action, which requires the `can_set_share_access` permission

Reply:
ok cool